ray-operator/config/samples/ray-cluster.complete.yaml

```diff
@@ -44,32 +38,7 @@ spec:
       spec:
         containers:
         - name: ray-head
-          image: rayproject/ray:1.11.0
-          env:
-          - name: CPU_REQUEST
-            valueFrom:
-              resourceFieldRef:
-                containerName: ray-head
-                resource: requests.cpu
-          - name: CPU_LIMITS
-            valueFrom:
-              resourceFieldRef:
-                containerName: ray-head
-                resource: limits.cpu
-          - name: MEMORY_LIMITS
-            valueFrom:
-              resourceFieldRef:
-                containerName: ray-head
-                resource: limits.memory
-          - name: MEMORY_REQUESTS
-            valueFrom:
-              resourceFieldRef:
-                containerName: ray-head
-                resource: requests.memory
-          - name: MY_POD_IP
-            valueFrom:
-              fieldRef:
-                fieldPath: status.podIP
```

Comment:
@DmitriGekhtman just curious if these env vars are deprecated and should no longer be used or if removing them will change Ray behavior? I'm using Ray 1.13.

Reply:
They're deprecated and can be safely removed.